src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyElementWrapper.java

```diff
@@ -0,0 +1,73 @@
+package com.wiley.autotest.selenium.elements.upgrade;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Wrap element with TeasyElement
+ */
+public class TeasyElementWrapper {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(TeasyElementWrapper.class);
```

Comment:
Could we use import here?

Reply:
replaced with Report class usage (to get rid of logger completely)